spec/lib/rbac/filterer_spec.rb

```diff
@@ -2509,6 +2512,9 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
 
       results = described_class.search(:class => Flavor, :user => other_user).first
       expect(results).to match_array [flavor_other]
+
+      results = described_class.search(:class => OrchestrationStack, :user => other_user).first
+      expect(results).to match_array [orchestration_stack_other]
```

Comment:
I would think the `other_user` would not see this. But I'm not unsure.

```suggestion
      expect(results).to match_array be_empty
```

Reply:
yes, why things like a `other_XXX ` are visible ?

Reply:
@lpichler Tenant mapping is disabled for objects that belong to ```cloud_tenant_other```